explorer/interpreter/action_stack.h

```diff
@@ -118,10 +120,12 @@ class ActionStack {
   // Suspends execution of the currently-executing continuation.
   auto Suspend() -> ErrorOr<Success>;
 
+  void Pop() { todo_.Pop(); }
+
  private:
   // Pop any ScopeActions from the top of the stack, propagating results as
   // needed, to restore the invariant that todo_.Top() is not a ScopeAction.
-  void PopScopes();
+  auto PopScopes() -> std::list<std::unique_ptr<Action>>;
```

Comment:
Just as a default choice, can you use `vector` instead of `list`? (unless there's a particular reason for the choice)

Reply:
I need push_back and push_front!